lib/matplotlib/offsetbox.py

```diff
@@ -569,14 +570,16 @@ class DrawingArea(OffsetBox):
     """
     The DrawingArea can contain any Artist as a child. The DrawingArea
     has a fixed width and height. The position of children relative to
-    the parent is fixed.
+    the parent is fixed. The children can be  clipped at the
```

Comment:
And if you have a change set, please open a PR.

On Mon, May 25, 2015 at 11:25 PM Hassan Kibirige <notifications@github.com>
wrote:

> In lib/matplotlib/offsetbox.py
> <https://github.com/matplotlib/matplotlib/pull/4466#discussion_r31002511>:
>
> > @@ -569,14 +570,16 @@ class DrawingArea(OffsetBox):
> >      """
> >      The DrawingArea can contain any Artist as a child. The DrawingArea
> >      has a fixed width and height. The position of children relative to
> > -    the parent is fixed.
> > +    the parent is fixed. The children can be  clipped at the
>
> Well, I rebased and pushed already.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4466/files#r31002511>.
>